scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -191,21 +206,68 @@ _deploy_deb()
 
 _install_user_mode_centos()
 {
-    _deploy_rpm "intel*.rpm"
-    if [[ $? -ne 0 ]]; then
-        echo "ERROR: failed to install rpms $cmd error"  >&2
-        echo "Make sure you have enough disk space or fix the problem manually and try again." >&2
-        exit $EXIT_FAILURE
+    if [[ "$INSTALL_DRIVER_VERSION" == "21.29.20389" ]]; then
+        CMDS=("dnf update --refresh"
```

Comment:
`dnf update` is an alias for `dnf upgrade` and will modify user's environment. This behavior can be considered undesirable.

Reply:
https://github.com/intel-gpu/documentation/issues/19